features/__snapshots__/task_validate_image.snap

```diff
@@ -491,3 +491,552 @@ true
   "TEST_OUTPUT": "{\"timestamp\":\"${TIMESTAMP}\",\"namespace\":\"\",\"successes\":1,\"failures\":0,\"warnings\":0,\"result\":\"SUCCESS\"}"
 }
 ---
+
+[Strict with failures:outcome - 1]
+{
+  "success": false,
+  "components": [
+    {
+      "name": "",
+      "containerImage": "${REGISTRY}/acceptance/does-not-exist",
+      "source": {},
+      "violations": [
+        {
+          "msg": "Image URL is not accessible: HEAD http://${REGISTRY}/v2/acceptance/does-not-exist/manifests/latest: unexpected status code 404 Not Found (HEAD responses have no body, use GET for details)",
+          "metadata": {
+            "code": "builtin.image.accessible",
+            "description": "The image URL is available and accessible.",
+            "title": "Image URL is accessible"
+          }
+        }
+      ],
+      "success": false
+    }
+  ],
+  "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEWgPQT7oJ2S9eTddeLwXKFuo6BPbh\ndMBvB8lZc+MCo5uf1PyAoq6/a/kFqNO2PuDguENYLPNqS4EwcePLbDQlEQ==\n-----END PUBLIC KEY-----\n",
```

Comment:
`${__known_PUBLIC_KEY}` ?

Reply:
Yeah the replacements we had did not work for JSON